src/Export/Provider/PriceBasedOnConfigurationProvider.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Provider;
+
+use FINDOLOGIC\FinSearch\Findologic\AdvancedPricing;
+use FINDOLOGIC\FinSearch\Struct\Config;
+use Shopware\Core\Checkout\Cart\Price\Struct\CalculatedPrice;
+use Shopware\Core\Checkout\Cart\Price\Struct\PriceCollection;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class PriceBasedOnConfigurationProvider
```

Comment:
Is this provider really needed?
Seems like a lot of overhead for something that could just be private methods. A lot of overhead, because it does not even need it's own services. The config is also loaded in the adapter.